content/en/docs/installing-on-linux/introduction/ha-configuration.md

```diff
@@ -45,26 +45,38 @@ You must create a load balancer in your environment to listen (also known as lis
 
 {{< tab "For users with good network connections to GitHub" >}}
 
-Download KubeKey from [GitHub Release Page](https://github.com/kubesphere/kubekey/releases/tag/v1.0.0) or use the following command directly.
+Download KubeKey from its [GitHub Release Page](https://github.com/kubesphere/kubekey/releases) or use the following command directly.
 
 ```bash
-wget https://github.com/kubesphere/kubekey/releases/download/v1.0.0/kubekey-v1.0.0-linux-amd64.tar.gz -O - | tar -xz
+curl -sfL https://get-kk.kubesphere.io | VERSION=v1.0.1 sh -
 ```
 
 {{</ tab >}}
 
 {{< tab "For users with poor network connections to GitHub" >}}
 
-Download KubeKey using the following command:
+Run the following command first to make sure you download KubeKey from the correct zone.
 
 ```bash
-wget -c https://kubesphere.io/download/kubekey-v1.0.0-linux-amd64.tar.gz -O - | tar -xz
+export KKZONE=cn
```

Comment:
The download command has added `KKZONE=cn`。
`export KKZONE=cn` should be executed before the kk command.

Reply:
Yes, I am aware of that. Is there a problem with the description here?